### PR TITLE
add release instructions to contributing.md

### PR DESCRIPTION
**Pull request by patflynn**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -83,6 +83,53 @@ A number of issues in the
...
+    git commit -a -m "Set plugin version to 1.0"
+    ```
+
+1. Push your change.
+
+    ```
+    git push origin release_v1.0
+    ```
+
+1. Create a PR and send your change for review with another team member. Label it with the 'Release' tag.
```
Comment (elharo) [inline CONTRIBUTING.md]: to another team member

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -83,6 +83,53 @@ A number of issues in the
...
+1. Merge when approved.
+1. Now we need to return to a snapshot release. Follow the same flow but update the version number to something like:
+
+    ```
+        version = '1.1-SNAPSHOT'
+    ```
+
+1. Commit your changes, push, send a PR, and merge.
+1. The final step is to go to label the release in Github.
```
Comment (elharo) [inline CONTRIBUTING.md]: delete "go to"

**Comment by elharo** [comment]
What do you think about making this a separate document? This isn't really for contributors.

**Comment by patflynn** [comment]
PTAL

**Comment by elharo** [comment]
lgtm

**Pull request merged**